docs/user-guide/backwards-compatibility.md

```diff
@@ -0,0 +1,12 @@
+# Backwards Compatibility
+
+Balancing the trade-offs of improving a tool and the frustration these changes can cause is a difficult task. One key area in which this affects our users is in the replacement of one rule by another.
+
+The ESLint team is committed to making upgrading as easy and painless as possible. To that end, the team has agreed upon some guidelines for replacing rules in the future. The goal of these guidelines is to allow for improvements and changes to be made without breaking existing configurations.
+
+For the next year, the team has committed to not remove any rules in any releases of ESLint; however, the team will deprecate rules as needed. When a rule is deprecated, it means that:
+
+* The team will no longer work on the rule. This includes bug fixes, enhancements, and updates to the rule's documentation. Any issues/pull requests related to the deprecated rule will not be accepted and will be closed automatically with a message referring to the rule that replaces it.
```

Comment:
Again, there might not always be a replacement. You can end the sentence after "closed automatically."